### PR TITLE
Updated project structure. 

### PR DESCRIPTION
**Pull request by Perelyn-sama**
Foundry has undergone some changes since this repo was created. 

Since this repo is so iconic, I thought it would be better for the changes to present here so readers won't get confused.

I'm open to any corrections you feel are needed in this PR.^-^

### CODE REVIEW

File: remappings.txt [hunk 1]
```diff
@@ -0,0 +1,4 @@
+forge-std/=lib/forge-std/src/
+openzeppelin-contracts/=lib/openzeppelin-contracts/
+solmate/=lib/solmate/src/
+weird-erc20/=lib/solmate/lib/weird-erc20/src/
```
Comment (FredCoen) [inline remappings.txt]: why remap wierd erc20

**Review by FredCoen — COMMENTED** [review]
cheers for the contrib! lgtm, just one comment. Also did you run the tests?

File: remappings.txt (hunk 1 above)
Comment (Perelyn-sama) [reply remappings.txt]: I have no idea where that came from😂

I think it was when I ran `forge remapping > remapping.txt`. 

I'll go remove it now.

**Comment by Perelyn-sama** [comment]
Yeah, it does. I'll send a screenshot in a bit. 

Or you can clone my forked version on your machine and run it too.

File: remappings.txt (hunk 1 above)
Comment (Perelyn-sama) [reply remappings.txt]: removed.

**Comment by FredCoen** [comment]
awesome thanks @Perelyn-sama

**Pull request merged**